Settings.hs

```diff
@@ -13,7 +15,9 @@ import Yesod.Default.Config
 import Yesod.Default.Util
 import Data.Text (Text)
 import Data.Yaml
+#if !(MIN_VERSION_base(4,8,0))
 import Control.Applicative
```

Comment:
Does this import no longer work on new versions of base?  It's redundant, but so what?